docs/onPrePush.md

```diff
@@ -0,0 +1,44 @@
+---
+title: onPrePush
+sidebar_label: onPrePush
+---
+
+The `onPrePush` callback is called before sending an object pack to remote and can be used to abort the push action.
+
+This callback is implemented as an equivalent to the canonical git `pre-push` hook. An `onPrePush` function is passed an object containing information about the target remote and `url`, local and remote `ref` name and the commit oids. This function must return `true` if the push action is to be aborted and `false` if not.
```

Comment:
I would reverse the values. I know that on Linux exit code 0 is all fine, but in JavaScript true is all fine, and false is that something is wrong. So this is not very intuitive.